recipes/weave/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "weave" %}
+{% set version = "0.15.0" %}
+{% set sha256 = "4b7ac15abe2d1c0901c33db79d4bdcad8f2261198da36a48707b0023bca72f83" %}
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py3k]  # Python2-only package
+  script: python setup.py install --record record.txt
+
+requirements:
+  build:
+    - python <3
+    - numpy
+  run:
+    - python <3
+    - numpy
+    - bsddb  # [not win]
+
+test:
+  imports:
+    - weave
+  commands:
+    - python -c "import weave;assert weave.inline('int x=42; return_val=x;') == 42"
+
+about:
+  home: https://pypi.python.org/pypi/weave
+  license: BSD
```

Comment:
Can you update this to `BSD 3-Clause` since [scipy has this license](https://github.com/conda-forge/scipy-feedstock/blob/master/recipe/meta.yaml#L47) and [weave is distributed under the same licence](https://github.com/scipy/weave/blob/master/LICENSE.txt#L1)